static/bidder-info/colossus.yaml

```diff
@@ -0,0 +1,12 @@
+maintainer:
+  email: "huddled.masses1650@gmail.com"
```

Comment:
I'm curious why you are using a gmail account as a contact here instead of an email with `@huddledmasses.com` or `@colossusssp.com`.

Reply:
@SyntaxNode Because maintanace is beeing done by outsource, so for any questions related to adapters github register email address is used

Reply:
@HuddledMasses Gotcha. Our concern over the gmail account is that it does't have good optics as being officially provided by HuddledMasses and if the contractor changes in the future, this support email address may be forgotten about. It's important to us because we rely on these contacts long term for support.

Would you consider creating a `@huddledmasses.com` and forward it to the this gmail address? That leaves HuddledMasses in control of the contact flow and can change it in the future if needed.

Reply:
I've added support@huddledmasses.com email